ember-cli-build.js

```diff
@@ -5,12 +5,12 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     babel: {
-      plugins: ['transform-object-rest-spread']
+      plugins: ['@babel/plugin-proposal-object-rest-spread']
```

Comment:
should this be in index.js instead? I think you also only need it in either babel or ember-cli-babel?

Reply:
It's a good question, I only made minimal changes to get Babel 7 working. Since this works, my guess is that the addon is not actually using object rest spread, it is only being used in the dummy app and/or tests, and thus does not need to be specified in index.js

I'm also not sure why the original code had both 'babel' and 'ember-cli-babel' keys, but since I'm not sure of the reason, I kept both.

Reply:
`babel` is for configuring Babel itself, like passing plugins.
`ember-cli-babel` is for configuring `ember-cli-babel` itself, like what file extensions to process.

See https://github.com/babel/ember-cli-babel#options.

`index.js` controls the compilation of the `addon` tree (among other things).
`ember-cli-build.js` controls the compilation of the `tests/dummy` app.

Reply:
So I should remove `plugins` from `ember-cli-babel` but I still need both keys, correct?

Reply:
Just checking in, do you feel like you're happy enough w/ this for me to merge it?

Reply:
As it works, I'm fine with it. I was waiting for a response from @buschtoens